coreneuron/mech/modfile/pattern.mod

```diff
@@ -116,6 +116,7 @@ void pattern_stim_setup_helper(int size, double* tv, int* gv, _threadargsproto_)
 	info->size = size;
 	info->tvec = tv;
 	info->gidvec = gv;
+	artcell_net_send ( _tqitem, -1, (Point_process*) _nt->_vdata[_ppvar[1*_STRIDE]], t +  0.0 , 1.0 ) ;
```

Comment:
A comment would be helpful. Is this needed for initialization after restore?

Reply:
I am testing it now.

Reply:
Done